flutter_appauth/example/lib/main.dart

```diff
@@ -175,7 +175,9 @@ class _MyAppState extends State<MyApp> {
       final AuthorizationTokenResponse result =
           await _appAuth.authorizeAndExchangeCode(
         AuthorizationTokenRequest(_clientId, _redirectUrl,
-            serviceConfiguration: _serviceConfiguration, scopes: _scopes),
+            serviceConfiguration: _serviceConfiguration,
+            scopes: _scopes,
+            preferEphemeralSession: true),
```

Comment:
Rather than forcing to be true in the existing example (which would also make it a bit out of sync with the readme), could you leave the original code as is and add another button (only visible on iOS) to the example that would then invoke the code where `preferEphemeralSession` is set to true?